airflow/providers/google/cloud/operators/automl.py

```diff
@@ -240,9 +240,9 @@ class AutoMLBatchPredictOperator(BaseOperator):
         written. If a dict is provided, it must be of the same form as the protobuf message
         `google.cloud.automl_v1beta1.types.BatchPredictOutputConfig`
     :type output_config: Union[dict, ~google.cloud.automl_v1beta1.types.BatchPredictOutputConfig]
-    :param params: Additional domain-specific parameters for the predictions, any string must be up to
+    :param prediction_params: Additional domain-specific parameters for the predictions, any string must be up to
```

Comment:
```suggestion
    :param prediction_params: Additional domain-specific parameters for the predictions, any string must be up to
```
This line is too long

Reply:
Thank you, last commit shortened these lines